lib/modulesync/cli.rb

```diff
@@ -7,6 +7,14 @@
 
 module ModuleSync
   module CLI
+    def self.prepare_options(cli_options, **more_options)
+      options = CLI.defaults
+      options.merge! Util.symbolize_keys cli_options
```

Comment:
I feel like braces here would be helpful. In general I'm not fan of skipping them except when you do DSL style things.

Reply:
Done.